src/initrd-cryptsetup.path

```diff
@@ -38,4 +41,7 @@ DirectoryNotEmpty=/run/systemd/ask-password
 WantedBy=sysinit.target
 
 [X-SystemdTool]
-# not used
+
+# provision disk tables in initramfs
```

Comment:
Is this because of the above note to remove `sd-encrpyt` from HOOKS?

Reply:
not sure what you mean